drape_frontend/message_subclasses.hpp

```diff
@@ -1151,21 +1151,41 @@ class DrapeApiFlushMessage : public Message
   TProperties m_properties;
 };
 
-class SetCustomSymbolsMessage : public Message
+class AddCustomSymbolsMessage : public Message
 {
 public:
-  explicit SetCustomSymbolsMessage(CustomSymbols && symbols)
+  explicit AddCustomSymbolsMessage(CustomSymbols && symbols)
     : m_symbols(std::move(symbols))
   {}
 
-  Type GetType() const override { return Message::SetCustomSymbols; }
+  Type GetType() const override { return Message::AddCustomSymbols; }
 
   CustomSymbols && AcceptSymbols() { return std::move(m_symbols); }
 
 private:
   CustomSymbols m_symbols;
 };
 
+class RemoveCustomSymbolsMessage : public Message
+{
+public:
+  RemoveCustomSymbolsMessage()
+    : m_removeAll(true)
+  {}
+
+  explicit RemoveCustomSymbolsMessage(MwmSet::MwmId const & mwmId)
+    : m_mwmId(mwmId)
+  {}
+
+  Type GetType() const override { return Message::RemoveCustomSymbols; }
+  bool NeedRemoveAll() const { return m_removeAll; }
+  MwmSet::MwmId const & GetMwmId() const { return m_mwmId; }
+
+private:
+  MwmSet::MwmId m_mwmId;
+  bool m_removeAll = false;
```

Comment:
Usually default initialization is used for default constructors. So, why not to:
```c++
RemoveCustomSymbolsMessage() = defaut;
explicit RemoveCustomSymbolsMessage(... mwmId): m_mwmId(mwmId), m_removeAll(false) {}
...
bool m_removeAll = true;
```
?